pkg/serviceaccounts/controllers/docker_registry_service.go

```diff
@@ -138,7 +138,7 @@ type DockerRegistryServiceController struct {
 }
 
 // Runs controller loops and returns immediately
-func (e *DockerRegistryServiceController) Run(workers int, stopCh <-chan struct{}) {
+func (e *DockerRegistryServiceController) Run(ctx context.Context, workers int) {
```

Comment:
pull stuff like this which are maintenance refactors without intent changes into a separate PR for easier review and merge